sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -156,6 +156,13 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
         },
       },
     };
+
+    // TODO: This type isn't part of the NodeAuthOptions. Should it be? Reach out to the MSAL team.
+    if (options.disableAuthorityValidation) {
+      (configuration.auth as any).validateAuthority = !options.disableAuthorityValidation;
+    }
```

Comment:
This only works because accidentally MSAL passes `validateAuthority` to their core configurations. I am asking them internally if they should remain consistent across languages with this property in their public API.